app/Services/PostService.php

```diff
@@ -70,8 +73,11 @@ public function update($signup, $data, $transactionId)
     {
         $postOrSignup = $this->repository->update($signup, $data);
 
+        // Send to Blink unless 'dont_send_to_blink' is TRUE
+        $should_send_to_blink = ! (array_key_exists('dont_send_to_blink', $data) && $data['dont_send_to_blink']);
+
         // Save the new post in Customer.io, via Blink.
-        if (config('features.blink') && $postOrSignup instanceof Post) {
+        if (config('features.blink') && $postOrSignup instanceof Post && isset($data['dont_send_to_blink']) && $should_send_to_blink) {
```

Comment:
why do you need this check: `isset($data['dont_send_to_blink'])` since you also have `$should_send_to_blink` that should be enough.

Reply:
meant to remove that one!